### PR TITLE
update function app task help string

### PR DESCRIPTION
**Pull request by vincentdass**

### CODE REVIEW

File: Tasks/AzureFunctionAppContainerV1/Strings/resources.resjson/en-US/resources.resjson [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "loc.friendlyName": "Azure Function for container",
-  "loc.helpMarkDown": "[More information](https://aka.ms/azurefunctiononcontainerdeployreadme)",
-  "loc.description": "Update Function Apps with Docker containers",
+  "loc.friendlyName": "Azure Functions for container",
```
Comment (kmkumaran) [inline Tasks/AzureFunctionAppContainerV1/Strings/resources.resjson/en-US/resources.resjson]: Isn't the Azure branding "Azure function"? Why this change?

Comment (vincentdass) [reply Tasks/AzureFunctionAppContainerV1/Strings/resources.resjson/en-US/resources.resjson]: its `Azure Functions`. PM from Azure Functions team asked to change this.
